src/components/Card.jsx

```diff
@@ -0,0 +1,26 @@
+// Importing react.
+import React from 'react';
+// Importing scss.
+import '../scss/card.scss';
+// Importing PropTypes, defines the props for this component.
+import PropTypes from 'prop-types';
+
+function Card({ children }) {
```

Comment:
We should probably add a `className` prop if we want to extend this class style.

Reply:
Added the prop.